steam_network.gd

```diff
@@ -348,23 +350,18 @@ func _broadcast_p2p_packet(data: PoolByteArray, send_type: int = Steam.P2P_SEND_
 			_send_p2p_packet(peer_id, data, send_type, channel)
 
 func _read_p2p_packet():
-	var packet_size = Steam.getAvailableP2PPacketSize(0)
-
-	# There is a packet
-	if packet_size > 0:
-		
-		# Packet is a Dict which contains {"data": PoolByteArray, "steamIDRemote": CSteamID}
-		var packet = Steam.readP2PPacket(packet_size, 0)
-		
-		# or empty if it fails
-		if packet.empty():
-			push_warning("Steam Networking: read an empty packet with non-zero size!")
+	# Packet is a Dict which contains {"data": PoolByteArray, "steamIDRemote": CSteamID}
+	var packet = Steam.readP2PPacket(packet_size, 0)
```

Comment:
I might be misreading this, but will packet_size not be undeclared at this point if its removed? Looks like it might need to be passed as an argument from the new while loop in _process?

Reply:
Passed in packet_size as you mentioned